tensorflow/core/api_def/base_api/api_def_Bitcast.pbtxt

```diff
@@ -12,6 +12,26 @@ If `T` is smaller than `type`, the operator requires that the rightmost
 dimension be equal to sizeof(`type`)/sizeof(`T`). The shape then goes from
 [..., sizeof(`type`)/sizeof(`T`)] to [...].
 
+tf.bitcast can be used over tf.cast when you have to assign some unsigned data type.
+For example,
+ tf.bitcast(tf.cast(my_tensor, tf.int8), tf.uint8)
+
+Example,
+>>> import tensorflow as tf
+>>> x = [1., 2., 3.]
+>>> y = [0., 2., 3.]
+>>> equality =tf.equal(x,y)
+>>> equality_cast = tf.cast(equality,tf.float32)
+>>> equality_bitcast = tf.bitcast(equality_cast,tf.uint8)
+>>> sess = tf.Session()
+>>> print(sess.run(equality))
```

Comment:
modify to run in TF 2.0

Reply:
I have not used TF2.0 that much , so I have to first work around the changes first